PlaidLink.tsx

```diff
@@ -38,7 +38,11 @@ export const usePlaidEmitter = (LinkEventListener: LinkEventListener) => {
 
 
 export const openLink = async (props: PlaidLinkProps) => {
+  if (props.tokenConfig == null) {
+    console.log('The public_key is being deprecated. Learn how to upgrade to link_tokens at https://plaid.com/docs/#create-link-token')
```

Comment:
Would https://plaid.com/docs/link-token-migration-guide/ be a more suitable resource here?

Reply:
Great suggestion, I updated the link!